### ISSUE TITLE
`COPY TO` writable stream error when using Deno >= 1.33.3

### ISSUE THREAD

**Issue by swgn** [porsager/postgres#609]
I'm receiving an error when I use `COPY TO` streaming via `node:stream/promises`.

```
import { pipeline } from "node:streams/promises";

const file_data = await Deno.open("file.csv");
const file_data_stream = file_data.readable;

const query = await sql`copy
                    ${sql(source.load_table)}
                    (${sql(columns)})
                from stdin
                    csv
                    header
                    delimiter ','
                    encoding 'windows-1251'
                `.writable();

await pipeline(
    file_data_stream as unknown as NodeJS.ReadableStream, // Fix type issues
    query as unknown as NodeJS.WritableStream // Fix type issues
);
```

Works flawlessly with Deno@v1.33.2, however if I update Deno >= 1.33.3, I receive a funky error:

```
TypeError [ERR_INVALID_ARG_TYPE]ams[stream.length - 1]" property must be of type function. Received an instance of Writable
    at __node_internal_ (ext:deno_node/_stream.mjs:379:15)
    at popCallback (ext:deno_node/_stream.mjs:5084:7)
    at pipeline (ext:deno_node/_stream.mjs:5165:41)
```

I can see some previous issues related to using `COPY TO` with dynamic table names and columns, however this does not seem have been an issue prior to Deno@v1.33.3

---

As an attempted workaround, I also tried using the Deno native streams module by updating the final line of my code:

```
await file_data_stream.pipeTo(query)
```

For which I receive the following error:

```
TypeError: Failed to execute 'pipeTo' on 'ReadableStream': Argument 1 is not of type WritableStream.
```

Any help much appreciated 🙏

**Comment by porsager** [comment]
Please reopen if you're still having issues

**Comment by swgn** [comment]
✨ Fixed today with the release of [deno 1.35.2](https://github.com/denoland/deno/releases/tag/v1.35.2)
